packages/client/src/components/Footer/Footer.style.css

```diff
@@ -1,176 +1,223 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+footer,
+footer > * {
+  box-sizing: border-box;
+}
+
+footer,
+.footer-bg,
 .footer-container,
-.footer-content-section,
-.footer-copyright-section,
-.placeholder-text {
+.copyright-disclaimer-container,
+.find-information-container,
+.logo-socialmedia-container,
+.links-container,
+.social-media-logo-container {
   display: flex;
   align-items: center;
 }
 
+footer,
+.copyright-disclaimer-container,
+.links-container,
+.footer-bg {
+  flex-direction: column;
+}
+
 .footer-container,
-.footer-copyright-section {
-  width: 100vw;
+.links-container {
+  justify-content: space-between;
 }
 
-.footer-content-section,
-.footer-copyright-infosection {
-  width: 1220px;
+footer {
+  height: 180px;
+  justify-content: center;
+  background: linear-gradient(
+      180deg,
+      rgba(28, 56, 64, 0.7) 0%,
+      rgba(28, 65, 64, 0.8) 0.01%,
+      rgba(7, 27, 32, 0.5) 100%
+    ),
+    url('../../../public/assets/images/concrete-bw.png');
+  background-repeat: repeat;
+  mix-blend-mode: normal;
+}
+
+.footer-bg {
+  background-color: rgba(0, 0, 0, 0.3);
+  mix-blend-mode: normal;
 }
 
 .footer-container {
-  background-color: rgba(40, 48, 49);
-  flex-direction: column;
+  width: 1220px;
```

Comment:
How does this react in smaller sizes of the screen?
I also see a 613px below, why aren't these using percentages to adjust and then let the breakpoints change which percentage to use? 

That's at least what I am used to doing :)

Reply:
Sorry for answering this question for so long. 
1) smaller sizes start from 1300px, when a user gets this point he has 90vw width all the time. All screens wider than 1300px should have 1220px of the footer as this is the max size 
2) below this footer container I made some sizes from px to % that change their width depending on screen size and left one that always stays the same